pkg/controller/migplan/registry_health_failed.go

```diff
@@ -0,0 +1,129 @@
+package migplan
+
+import (
+	"github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
```

Comment:
need to order the imports https://godoc.org/golang.org/x/tools/cmd/goimports

Reply:
Done